tests/_sync/client.py

```diff
@@ -1,9 +1,14 @@
 from postgrest import SyncPostgrestClient
 
-REST_URL = "http://127.0.0.1:3000"
+REST_URL = "https://kzwoknoxuuvryajohvij.supabase.co/rest/v1/"
```

Comment:
This shouldn't be using a hosted instance because these could get paused or deleted, we should rely on the local docker instance instead.